assets/ca/ca.querytranslator.js

```diff
@@ -81,15 +81,14 @@ var caUI = caUI || {};
 				case 'between':
 					return prefix + '[' + escapeValue(ruleSet.value[0]) + ' TO ' + escapeValue(ruleSet.value[1]) + ']';
 				case 'begins_with':
-					return prefix + '"' + escapeValue(ruleSet.value) + '"*';
+					return prefix + '' + escapeValue(ruleSet.value) + '*';
```

Comment:
Don't need the empty string here.  But I'm still concerned about how this works with multiple words, it will be `ca_objects.FieldName:first second*` which will search for an exact match on `first` in the `FieldName` field and then `second*` anywhere.  Or am I mistaken?  Probably should dev box this case.